variants/metal-k8s-1.21/Cargo.toml

```diff
@@ -0,0 +1,39 @@
+[package]
+# This is the metal-k8s-1.21 variant.  "." is not allowed in crate names, but
+# we don't use this crate name anywhere.
+name = "metal-k8s-1_21"
+version = "0.1.0"
+edition = "2018"
+publish = false
+build = "build.rs"
+# Don't rebuild crate just because of changes to README.
+exclude = ["README.md"]
+
+[package.metadata.build-variant.image-layout]
+partition-plan = "unified"
+
+[package.metadata.build-variant]
+image-format = "raw"
```

Comment:
Let's also expand the OS image size to 4 GiB (two banks of 2 GiB) - since I expect to need some extra space for firmware, etc.